src/components/welcome/MyAccounts/useSafesSearch.ts

```diff
@@ -11,11 +11,13 @@ const useSafesSearch = (safes: (SafeItem | MultiChainSafeItem)[], query: string)
   const chains = useAppSelector(selectChains)
 
   useEffect(() => {
-    trackEvent({
-      category: OVERVIEW_EVENTS.SEARCH.category,
-      action: OVERVIEW_EVENTS.SEARCH.action,
-      label: query,
-    })
+    if (query) {
+      trackEvent({
+        category: OVERVIEW_EVENTS.SEARCH.category,
+        action: OVERVIEW_EVENTS.SEARCH.action,
+        label: query,
```

Comment:
I would not track the search query as the label as it can contain sensitive user information like address book names.

Reply:
ah I had this to debug the debounce, thanks!